dashboard/app/jobs/evaluate_rubric_job.rb

```diff
@@ -120,6 +173,38 @@ class EvaluateRubricJob < ApplicationJob
     # We gracefully just fail, here, and we do not file this exception
   end
 
+  RETRIES_ON_RATE_LIMIT = 3
+
+  # Retry on any reported rate limit (429 status) 'exponentially_longer' waits 3s, 18s, and then 83s.
+  retry_on TooManyRequestsError, wait: :exponentially_longer, attempts: RETRIES_ON_RATE_LIMIT do |job, error|
+    # Job arguments are always serializable, so we just pull out the hash
+    # and send it as context.
+    options = job.arguments.first
+
+    # Send it to honeybadger even though we are handling the error via retry
+    Honeybadger.notify(
+      error,
+      error_message: "Retrying rubric evaluation job due to rate limiting (429).",
+      context: options
+    )
+  end
+
+  RETRIES_ON_TIMEOUT = 2
+
+  # Retry just once on a timeout. It is likely to timeout again.
+  retry_on Net::ReadTimeout, Timeout::Error, wait: 10.seconds, attempts: RETRIES_ON_TIMEOUT do |job, error|
+    # Job arguments are always serializable, so we just pull out the hash
+    # and send it as context.
+    options = job.arguments.first
+
+    # Send it to honeybadger even though we are handling the error via retry
+    Honeybadger.notify(
+      error,
+      error_message: "Retrying rubric evaluation job due to timeout.",
+      context: options
+    )
```

Comment:
I don't want the timeouts to silently happen.

Reply:
hmm yes good question. let's do honeybadger so that we can get the details for now. If we get to where we know what info we want to pull out of these requests (or not), then we could change this to just log the info we need to cloudwatch (or just increment a simple counter) without clogging up honeybadger.